# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: 91f90a472a4c
# Full Hash: 91f90a472a4c55f06e898cabcdcc7d9a6ab80f04
# Author: Jan-Niklas Jaeschke <jjaschke@mozilla.com>
# Date: 2025-05-27 03:54:23
# Regressor Bug: 1957557
# File Overlap Count: 1
# Description:
#   Bug 1957557, part 5 - Create Text Fragments: Determine whether to use exact or range-based matching. r=dom-core,farre
#   
#   Range-based matching has to be used if
#    - there is a block boundary in the target range
#    - the range content is too long.
# ==============================================================================

diff -r 09653b9892a2 -r 91f90a472a4c modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Mon May 26 08:18:27 2025 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Mon May 26 08:18:28 2025 +0000
@@ -4859,6 +4859,11 @@
   value: 5
   mirror: always
 
+- name: dom.text_fragments.create_text_fragment.exact_match_max_length
+  type: uint32_t
+  value: 200
+  mirror: always
+
 - name: dom.textMetrics.actualBoundingBox.enabled
   type: RelaxedAtomicBool
   value: true
